lib/tasks/chattermill_reemit_events.rake

```diff
@@ -18,7 +18,7 @@ namespace :chattermill do
 
     puts "Searching failed events between #{from} and #{to}"
 
-    statuses = ['%502 Bad Gateway%', '%504 Gateway Time-out%', '%something went wrong (500)%']
+    statuses = ['%502 Bad Gateway%', '%504 Gateway Time-out%', '%something went wrong (500)%', '%default backend - 404%']
```

Comment:
can we just say `404`? Not sure the `default backend` option will be there

Reply:
It's no safe, because It would looks for any '404' in the payload.

Reply:
OK, fair enough